controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -218,7 +219,9 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(request ctrl.Reques
 		return ctrl.Result{}, err
 	}
 
-	policyconditions.Reset(r.Client, request.NamespacedName)
+	if instance.ObjectMeta.Annotations[nncp.TimestampLabelKey] != "" {
+		policyconditions.Reset(r.Client, request.NamespacedName)
+	}
```

Comment:
This will crash if it's not present, this has to be:
```golang
_, hasWebhookLabel := instance.ObjectMeta.Annotations[nncp.TimestampLabelKey]
if !hasWebhookLabel {
    policyconditions.Reset(r.Client, request.NamespacedName)
}
```

Reply:
ouch thanks, updated